lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -603,12 +603,15 @@ def test_svg_font_string(font_str, include_generic):
     text_count = 0
     for text_element in tree.findall(f".//{{{ns}}}text"):
         text_count += 1
-        font_info = dict(
+        font_style = dict(
             map(lambda x: x.strip(), _.strip().split(":"))
             for _ in dict(text_element.items())["style"].split(";")
-        )["font"]
+        )
+        font_size = font_style["font-size"]
+        font_family = font_style["font-family"]
 
-        assert font_info == f"{size}px {font_str}"
+        assert font_size == f"{size}px"
+        assert font_family == font_str
```

Comment:
We don't need the intermediate local variables from l.609/610.
```suggestion
        assert font_style["font-size"] == f"{size}px"
        assert font_style["font-family"] == font_str
```
is more concise.